src/Raven.Studio/wwwroot/App/views/resources/createDatabase.html

```diff
@@ -331,23 +342,23 @@ <h3>Backup Source Configuration</h3>
                             <label class="control-label">Region Name</label>
                         </div>
                         <div class="col-md-8">
-                            <label class="control-label text-info text-left" data-bind="text: awsRegionName"></label>
+                            <label class="control-label text-info text-left" data-bind="text: amazonS3().regionName"></label> 
```

Comment:
Can we have a dropdown like we have in ```Edit Periodic Backup```?

Reply:
done